src/js/components/CommonElement/TokenSelectorView.js

```diff
@@ -52,6 +52,13 @@ const TokenSelectorView = (props) => {
     })
   }
 
+  var shortName = (name) => {
```

Comment:
Short name please use format: Basic Attention Ne...., Please try to use css, not javascript, It'll help display full name in large screen